### PR TITLE
Resolved MUWM-3504 to show credit info

### PR DESCRIPTION
**Pull request by DannyNineNineNine**
Made changes to display credit info in the top-right corner of Course Cards (beneath meeting type).

**Additionally changed:**
- course title font to open sans
- offterm date to open sans
- positiong of offterm date message to above meeting times

**Used "javerage" to show:**
- typical course card
- one with an offterm date

### CODE REVIEW

**Comment by ezturner** [comment]
This is built on my-dev

**Comment by DannyNineNineNine** [comment]
Thanks @ezturner. Visually looks good. I'll need to find an offterm date and double-check. 
Then we'll hook it up to the credit info?

File: myuw/static/css/course_card.less [hunk 1]
```diff
@@ -72,7 +72,12 @@ c8a #473EC5;
         padding: 2px 4px 2px 4px;
         border: 1px solid #eee;
     }
-
+    
+	.course-credit-type {
```
Comment (klpittman) [inline myuw/static/css/course_card.less]: @DannyNineNineNine What is this class for? I don't see it referenced anywhere or used.

Comment (DannyNineNineNine) [reply myuw/static/css/course_card.less]: I'll check this when I'm back in the office on Thursday.

**Comment by DannyNineNineNine** [comment]
@klpittman about the class "course-credit-type": I'm not using it. I missed deleting it out. How can I revise and put into Git?

**Pull request closed without merging**